### DISCUSSION TITLE
Q: What is being imported _back_ into Krux from the wallet "coordinator"?

### DISCUSSION THREAD

**Discussion by david-bakin** [selfcustody/krux#61] [Q&A]
[This is not so much an issue as a question - I couldn't find a reference to any Krux discussion forum where I could ask it. BTW, if you don't have one yet I suggest starting a channel on Telegram which is where the SeedSigner / ColdCard / Raspiblitz / Specter / Sparrow and other projects hang out.]

The single-sig instructions (for Specter) proceed as follows:  Create mnemonic, export to wallet coordinator via QR, then _export from wallet coordinator back to Krux_ via QR.  Question is: What is being moved _back_ to the Krux that it doesn't already know from the mnemonic?

**Comment by ghost** [comment]
Hi! Good question. 

For single-key wallets, loading the wallet information is optional in terms of needing to check that receive addresses are valid or to sign PSBTs; in that case, it's assumed to be the `wpkh` script type. Where the import is more useful is when the wallet coordinator includes extra information beyond just the output descriptor in the wallet backup, such as how Specter Desktop also includes the device list and wallet name. Krux will keep the data in the format it was received when it generates its own QR code that you can scan or print, which in this case would let you load up a new Specter Desktop instance with the correct wallet metadata.

For multisig, the wallet import is more important obviously since it includes the cosigners which can't be inferred otherwise.

**Reply by ghost** [reply]
@david-bakin P.S.: I enabled Discussions and converted your issue into the first one under Q&A! Hopefully this will be a decent place for people to come and ask questions, propose ideas, etc.
